emgapianns/management/commands/import_analysis.py

```diff
@@ -162,10 +162,22 @@ def __find_existing_result_dir(self, secondary_study_accession, run_accession, v
         directory = os.path.join(self.rootpath, '2019')
         study_folder = self.__find_folder(directory, search_pattern=secondary_study_accession, recursive=True)
 
-        directory = os.path.join(study_folder, 'version_{}/'.format(version))
-        result_folder = self.__find_folder(directory, search_pattern=run_accession, maxdepth=3)
-        logging.info("Found the following result folder:\n{}".format(result_folder))
-        return result_folder
+        # find version_{} folder
+        result_folder = []
+        if len(study_folder) == 0:
+            sys.exit('Could not find result directory for: {}'.format(secondary_study_accession))
+        else:
```

Comment:
Code style:
the else statement is not required, the code will be interrupted if the `len(study_folder) == 0`